src/components/wb-playground/kyn-linked-div-wb/kyn-linked-div-wb.scss

```diff
@@ -0,0 +1,102 @@
+@use '../../../common/scss/global.scss';
+@use '~@kyndryl-design-system/shidoka-foundation/scss/mixins/typography.scss';
+
+:host {
+    width: 100%;
+    display: inline-block;
+}
+
+#linked-div {
+    max-width: 35rem;
+    position: relative;
+    overflow: hidden;
+    box-shadow: 0px 2px 8px 0px rgba(110, 109, 109, 0.1);
+    border-radius: 5px;
+    background-color: white; // Ensure the initial background is white
+
+    &::before {
+        content: '';
+        position: absolute;
+        top: 0;
+        left: -100%;
+        width: 100%;
+        height: 100%;
+        background-color: var(--hover-color, var(--kd-color-spruce-70));
+        transition: left 0.3s ease-in-out;
+        z-index: 0;
+    }
+
+    &:hover::before {
+        left: 0;
+    }
+}
+
+kd-link {
+    display: block;
+    width: 100%;
+    height: 100%;
+    position: relative;
+    z-index: 1;
+    text-decoration: none;
+    color: inherit;
+
+    &::after {
+        content: none;
+    }
+
+    &:hover {
+        text-decoration: none;
+        color: white;
+
+        &::after {
+            content: none;
+        }
+    }
+}
+
+#linked-div-content {
+    padding: 2rem 5rem 3rem 2rem;
+    position: relative;
+    z-index: 1;
+}
+
+#secondary-header {
+    @include typography.type-ui-02;
+    margin: 0;
+    padding: 0;
+    color: var(--kd-color-darkstone-50);
+    font-weight: 500;
+    text-transform: uppercase;
+
+    &::after {
+        content: none;
+    }
+}
+
+#main-header {
+    @include typography.type-headline-06;
+    margin: 1.5rem 0 3.25rem;
```

Comment:
We generally use `px` for spacing, so it remains consistent when font scales. There's also have a Spacing module in Foundation, though I don't typically use it since it maps directly to px.